Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.10.11
+
+# Install JupyterHub and dependencies
+RUN pip3 --disable-pip-version-check install --no-cache-dir \
+    jupyterhub==3.1.1 \
+    notebook==6.5.2 \
+    jupyterlab==3.5.3 \
+    jupyterlab-server==2.16.0
+
+# Install AlphaD3M and dependencies
+# RUN pip3 --disable-pip-version-check install --no-cache-dir \
+# 	alpha-automl
+
+ADD . /alpha-automl/
+WORKDIR /alpha-automl/
+ARG BUILD_OPTION='full'
+RUN pip3 install -e .[$BUILD_OPTION]
```

Comment:
This build does not work if we pass an empty string or do not pass the `BUILD_OPTION` parameter, i.e., we can't build an image of the default lightweight version. Maybe we should check if the parameter was set, and if not, just run only `pip3 install -e .`